install/Install.ps1

```diff
@@ -0,0 +1,88 @@
+# PowerShell Script to Install Spice CLI on Windows
+
+$spiceBin = ".spice\bin"
+$spiceCliInstallDir = Join-Path $HOME $spiceBin
+$spiceRepoName = "spiceai"
+$spiceOrgName = "spiceai"
+$spiceCliFileName = "spice.exe"
+$spiceCliFullPath= Join-Path $spiceCliInstallDir $spiceCliFileName
+
+# Ensure the installation directory and auth file exist
+New-Item -Path $spiceCliInstallDir -ItemType Directory -Force
+
+function Get-LatestRelease {
+    $url = "https://api.github.com/repos/$spiceOrgName/$spiceRepoName/releases/latest"
+    $releaseInfo = Invoke-RestMethod -Uri $url
+    return $releaseInfo.tag_name
+}
+
+function Verify-Supported {
+    $osInfo = Get-CimInstance Win32_OperatingSystem
+    $systemInfo = Get-CimInstance CIM_ComputerSystem
+
+    $currentOs = $osInfo.Caption
+    $currentArch = $systemInfo.SystemType
+
+    if ($currentOs -notlike "*Windows*") {
+        Write-Host "This script is only supported on Windows operating systems."
+        exit 1
+    }
+
+    Write-Host "System architecture is $currentArch"
+}
+
+# Function to download and install Spice CLIv
+function Download-And-Install-Spice {
+    Write-Host "Checking the latest Spice version..."
+
+    $latestReleaseTag = Get-LatestRelease
+    $arch = "x86_64"
```

Comment:
Currently have only x86_64 release binaries only. We will update this when we have separate binaries for different architectures.